### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by kgauthreaux**

### CODE REVIEW

File: forms/enrollment/naccid-enrollment-form_questions_and vars.csv [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+4f,Gender--Two-spirit,ENRLGENTWOSPIR,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4g,Gender--I use a different term (specify),ENRLGENOTH,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4g1,Gender--I use a different term specify,ENRLGENOTHX,Conditional,text,String,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4h,Gender--Don't know,ENRLGENDKN,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
+4i,Gender--Prefer not to answer,ENRLGENNOANS,Always,1 or blank,Integer,"I know this var is >10 characters, but I think I'd prefer them to match the UDSv4 A1 variables rather than meet our made-up character limit. Jessica agrees"
```
Comment (hewanadun) [inline forms/enrollment/naccid-enrollment-form_questions_and vars.csv:19]: 4a - 4i fields are not required but marked as "Always"?

File: forms/enrollment/naccid-enrollment-form_error_checks.csv [hunk 2]
```diff
@@ -1,42 +1,63 @@
...
+enrl-p-1001,1001,Error,enrl,FRMDATE_ENRL,Plausibility,FRMDATE_ENRL earliest date allowed,FRMDATE_ENRL must not be from date before form was released,Header variable FRMDATE_ENRL must not be from date before form was released,IF FRMDATE_ENRL before (03-01-2024),n/a,n/a,Yes,n/a
+enrl-p-1002,1002,Error,enrl,PREVENRL,Plausibility,PREVENRL must align with ENRLTYPE,If ENRLTYPE=2 then PREVENRL must equal 1,If Q1. ENRLTYPE (Assign NACCID or transfer?)=2 (Transfer) then Q6. PREVENRL (Previously enrolled at an ADRC?) must be equal to 1 (Yes),IF ENRLTYPE=2 and PREVENRL in (0 or 9),n/a,"ENRLTYPE, PREVENRL",Yes,n/a
+enrl-p-1003,1003,Error,enrl,OLDADCID,Plausibility,OLDADCID cannot equal ADCID,OLDADCID cannot be the same as ADCID,Q6a. OLDADCID (Previous ADCID) cannot be the same as Header variable ADCID,IF OLDADCID =ADCID,n/a,"OLDADCID, ADCID",Yes,n/a
+enrl-p-1004,1004,Error,enrl,PTIDCONF,Plausibility,PTIDCONF cannot equal PTID,PTIDCONF cannot be the same as PTID,Q8. PTIDCONF (Confirm PTID) cannot be the same as Header variable PTID,IF PTIDCONF=PTID,n/a,"PTIDCONF, PTID",Yes,n/a
```
Comment (hewanadun) [inline forms/enrollment/naccid-enrollment-form_error_checks.csv:63]: I think the check should be PTIDCONF must be equal PTID ??

**Review by hewanadun — CHANGES_REQUESTED** [review]
- Please check the comments and make the changes
- Can you please format the error_checks CSV to be compatible with the [REDCap import format](https://uwnetid.sharepoint.com/:x:/r/sites/og_national_alzheimers_coordinating_center/Shared%20Documents/Projects/Data%20Quality/NACCID%20Enrollment%20Form/naccid-enrollment-form_error_checks_redcap_import.csv?d=w7b01313450c74538bcac8aa80725447c&csf=1&web=1&e=HZlVpP) 
- Delete the naccid-enrollment-form_questions_and_variables.csv file since it is uploaded with a different name.
- Variables PTXFER, NEWADCID, LSTVISDATE are still in REDCap form but removed from vars list and error checks, is that a mistake or do they need to be removed from the REDCap form?

File: forms/enrollment/naccid-enrollment-form_questions_and vars.csv (hunk 1 above)
Comment (kgauthreaux) [reply forms/enrollment/naccid-enrollment-form_questions_and vars.csv:19]: At least one of them is required to be checked. This is how we wrote it for UDSv4 as well. We didn't really know how to represent that.

File: forms/enrollment/naccid-enrollment-form_error_checks.csv (hunk 2 above)
Comment (kgauthreaux) [reply forms/enrollment/naccid-enrollment-form_error_checks.csv:63]: Oops, yes, I completely reversed the language in my head. The error should be if they are not equal.

**Comment by kgauthreaux** [comment]
@hewanadun thanks! See below:
Please check the comments and make the changes 
- **DONE**
Can you please format the error_checks CSV to be compatible with the [REDCap import format](https://uwnetid.sharepoint.com/:x:/r/sites/og_national_alzheimers_coordinating_center/Shared%20Documents/Projects/Data%20Quality/NACCID%20Enrollment%20Form/naccid-enrollment-form_error_checks_redcap_import.csv?d=w7b01313450c74538bcac8aa80725447c&csf=1&web=1&e=HZlVpP) 
- **I thought I did do this? Can you point out what is different if it is not what you expected?**
Delete the naccid-enrollment-form_questions_and_variables.csv file since it is uploaded with a different name. 
- **DONE**
Variables PTXFER, NEWADCID, LSTVISDATE are still in REDCap form but removed from vars list and error checks, is that a mistake or do they need to be removed from the REDCap form?
**-They are currently hidden from the survey in the REDCap form so they will not show and are not required since they are no longer a part of the form. I think Laura left them there just in case we needed them later. But if you think they should be deleted I can do that.**

File: forms/enrollment/naccid-enrollment-form_questions_and vars.csv (hunk 1 above)
Comment (hewanadun) [reply forms/enrollment/naccid-enrollment-form_questions_and vars.csv]: Wouldn't it be more appropriate to mark these variables as "Conditional"? The error check enrl-m-025 will capture the condition that at least one of these need to be filled. If we mark them as required fields form will be rejected unless all of them are checked.

**Comment by hewanadun** [comment]
> * **I thought I did do this? Can you point out what is different if it is not what you expected?**
>   Delete the naccid-enrollment-form_questions_and_variables.csv file since it is uploaded with a different name.

**Old file is deleted now, it wasn't before**

>   Variables PTXFER, NEWADCID, LSTVISDATE are still in REDCap form but removed from vars list and error checks, is that a mistake or do they need to be removed from the REDCap form?
>   **-They are currently hidden from the survey in the REDCap form so they will not show and are not required since they are no longer a part of the form. I think Laura left them there just in case we needed them later. But if you think they should be deleted I can do that.**

**We can keep them in the form if they are not visible and no data will be entered.**

For error check enrl-m-025 please put only one variable in "var_name" column and rest in "comp_vars" column.

**Review by hewanadun — CHANGES_REQUESTED** [review]
Please check my comments above. Thanks!

**Comment by kgauthreaux** [comment]
@hewanadun okay I believe all requested changes that we talked about today have been made!

**Review by hewanadun — APPROVED** [review]
Looks good. Thanks for the updates.

**Pull request merged**